car_manager.py

```diff
@@ -0,0 +1,38 @@
+import random
+from turtle import Turtle
+COLORS = ["red", "orange", "yellow", "green", "blue", "purple"]
+STARTING_MOVE_DISTANCE = 5
+MOVE_INCREMENT = 10
+
+
+class CarManager(Turtle):
+
+    def __init__(self):
+        super().__init__()
+        self.all_cars = []
+        self.create_car()
+        self.speed = 10
+
+    def create_car(self):
+        random_chance = random.randint(1,6)
+        if random_chance == 1:
```

Comment:
This is a minor one and is really only for code aesthetics. But you can avoid the nesting if you just make it return empty if the random_chance is not equal to 1 so that the code only executes if the random_chance is equal to 1. But as I said, this is a very small one and honestly just my personal preference.